pkg/api/asr.go

```diff
@@ -133,6 +134,14 @@ func (s *service) GetContainerInfo(ctx context.Context, _ *task.ContainerInfoReq
 
 	ci := task.ContainersInfo{}
 	for name, container := range containers {
+		var labels string
+		labelsJson, err := json.Marshal(container.Spec.Labels)
+		if err == nil {
+			labels = string(labelsJson)
+		} else {
+			log.Debug().Msgf("error marshalling labels: %v", err)
```

Comment:
```suggestion
			log.Info().Msgf("error marshalling labels: %v", err)
```
Let's make it Info or Error?